components/sysutils/coreutils/Makefile

```diff
@@ -46,21 +52,25 @@ PATCH_LEVEL=		0
 # Prevent gcc register spill error on 32-bit SPARC
 gcc_OPT.sparc.32 =	-O1
 
-CONFIGURE_PREFIX	 =	/usr/gnu
-CONFIGURE_OPTIONS	+=	--libdir=/usr/lib
+# Recreate configure and Makefile to pick up changes in .../src/local.mk
+COMPONENT_PREP_ACTION = (cd $(@D) ; autoreconf -f && automake)
+
+CONFIGURE_PREFIX	 =	$(GNUDIR)
+CONFIGURE_BINDIR.64	 =	$(GNUBIN)
+CONFIGURE_OPTIONS	+=	CPPFLAGS=" -I$(USRINCDIR)/gmp"
+CONFIGURE_OPTIONS	+=	--libdir=$(USRLIBDIR.$(BITS))
```

Comment:
Likely, also should set libexecdir.
Also, are there any libstdbuf.so consumers?